File: src/test/java/com/google/devtools/java2csharp/testclasses/admin/ContentArea.java
package com.google.devtools.java2csharp.testclasses.admin;

public class ContentArea {
	private int id;

	private String name;

	private boolean managed;

	protected ContentArea() {
		super();
	}

	public ContentArea(int id, String name, boolean managed) {
		super();
		this.id = id;
		this.name = name;
		this.managed = managed;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public boolean isManaged() {
		return managed;
	}

	public void setManaged(boolean managed) {
		this.managed = managed;
	}
}

File: src/test/resources/expectedfolder/admin/ContentArea.cs
using System;

namespace com.google.devtools.java2csharp.testclasses.admin
{
	public class ContentArea
	{
		private int id;

		private string name;

		private bool managed;

		protected internal ContentArea()
			: base()
		{
		}

		public ContentArea(int id, string name, bool managed)
			: base()
		{
			this.id = id;
			this.name = name;
			this.managed = managed;
		}

		public virtual int Id
		{
			get
			{
				return id;
			}
			set
			{
				int id = value;
				this.id = id;
			}
		}

		public virtual string Name
		{
			get
			{
				return name;
			}
			set
			{
				string name = value;
				this.name = name;
			}
		}

		public virtual bool Managed
		{
			get
			{
				return managed;
			}
			set
			{
				bool managed = value;
				this.managed = managed;
			}
		}
	}
}
